src/aiida_quantumespresso/workflows/pw/bands.py

```diff
@@ -229,7 +229,6 @@ def run_scf(self):
         inputs.metadata.call_link_label = 'scf'
         inputs.pw.structure = self.ctx.current_structure
         inputs.pw.parameters = inputs.pw.parameters.get_dict()
-        inputs.pw.parameters.setdefault('CONTROL', {})['calculation'] = 'scf'
```

Comment:
If we are removing this, maybe we move line 231 to the `if self.ctx.current_number_of_bands:` block, since then we only unwrap-and-wrap the parameters if we really need to update it.

Reply:
Want to still add this change @mbercx ? Then this can be merged

Reply:
I missed this comment somehow. On it! 🚀